packages/syft/src/syft/service/message/message_stash.py

```diff
@@ -26,6 +27,8 @@
 )
 StatusPartitionKey = PartitionKey(key="status", type_=MessageStatus)
 
+OrderByCreatedAtTimeStampPartitionKey = PartitionKey(key="created_at", type_=DateTime)
```

Comment:
If the above occurs for all the stashes, maybe we could make the time order retrieval , `inherent` in the base stash.
For example, when a user tries to get a specific project.

They would initially view the projects


alice_domain_client.projects

In the second cell, when they index the project , based on the first cell

alice_domain_client.projects[1]

if the insertion order would have changed maybe we could get different one in the second cell?

Reply:
SUGGESTION:
Since we have

1.) Dictionary  --> In-memory store
MongoClient --> Containers

We could use

1.) Ordered Dictionary --> InMemory
2.) MongoClient --> We could modify the query API to sort by 
`db.myCollection.find().sort({_id: 1})`
I googled that, _id have inherently a timestamp field.


But not sure, if updates would change the `_id`

Reply:
@rasswanth-s , currently, the Dictionary and Mongo databases return values according to the order of insertion by default. The problem comes with the SQLite store as we don't use auto-incremental ids or store creation timestamp or any column to indicate order for all syft objects. And therefore we're passing order_by on case by case basis. Maybe we can have `created_at` as a default field to all syft objects and then we can always maintain that order by default. Let me once verify the uses of SyftObject.

Reply:
I see, Yeah , Shubham, we could also loop in Madhava also for feedback